manifests/Makefile

```diff
@@ -0,0 +1,39 @@
+ARGO_ROLLOUTS_VERSION := v0.10.2
```

Comment:
```suggestion
ARGO_ROLLOUTS_VERSION := $(shell yq r ../test/cicd.yaml argoRollouts.version)
```

Reply:
After some changes to the Makefile, now I think it is better to use `manifests/upstream` as a single source of truth for generating manifests. So I'm hard-coding version of Argo Rollouts manifests in `upstream/argo-rollouts.yaml` and `upstream/crd/argo-rollouts.yaml`.